src/programming/test/unit/patterns/test_dependency_injection.py

```diff
@@ -2,12 +2,9 @@
 from dataclasses import dataclass
 from logging import Logger
 
-from BL_Python.programming.patterns.dependency_injection import (
-    BatchModule,
-    LoggerModule,
-)
-from BL_Python.programming.str import get_random_str
 from injector import Injector
+from Ligare.programming.patterns.dependency_injection import BatchModule, LoggerModule
+from Ligare.programming.str import get_random_str
```

Comment:
Really, ruff? The line becomes 3 characters shorter and it therefore has to be reflowed?

Reply:
Snark aside, it was hugely helpful to do the reformatting in a different commit - awesome job.

Reply:
That's how it do be when only the tools have opinions.